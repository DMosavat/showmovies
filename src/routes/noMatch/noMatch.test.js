import React from 'react';
import { shallow } from 'enzyme';
import NoMatch from './index'

describe('noMatch Test', () => {

    describe('<NoMatch />', () => {

        it('should render the form', () => {
            const wrapper = shallow(<NoMatch />);
            expect(wrapper.html()).toMatchSnapshot();
        })
    
    })

})