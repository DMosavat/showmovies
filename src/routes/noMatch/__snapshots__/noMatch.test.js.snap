// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`noMatch Test <NoMatch /> should render the form 1`] = `"<main style=\\"height:768px\\"><div class=\\"container\\" style=\\"margin-top:150px\\"><div class=\\"row\\"><div class=\\"col-8\\"><a href=\\"/\\" style=\\"color:#FFF\\"> Home </a><h1>Page not found!</h1></div></div></div></main>"`;
